profiles/common/modules/custom/ecas/ecas_import_users/ecas_import_users.module

```diff
@@ -308,23 +309,19 @@ function ecas_import_users_show_user_content() {
 function ecas_import_users_show_user_content_form($form, &$form_state, $arguments) {
   $filters = '';
 
-  // Extract will set $names, $dg, $search_by, $country and $mail.
```

Comment:
I do not understand why here you hard code the values of names, dg, search_by, country and mail.
These are potentially set in the $import_arguments here
https://github.com/ec-europa/platform-dev/blob/master/profiles/common/modules/custom/ecas/ecas_import_users/ecas_import_users.module#L290

Reply:
I don't understand it either but I think that's out of the scope of this PR. We merely rewrote this lines in order to be compliant with QA without altering its logic. @drupol made this changes he can give you more information in the matter.

Reply:
Yes i understand, but the problem is that apparently the rewriting does alter the logic. The new code sets the value of names, dg, search_by, country and mail, and doing this they may override existing values.

Reply:
The current code sets the values to the variables only if they are not present, just as it did before as far as I understand.

Reply:
OK after checking i agree!